### PR TITLE
Added Add Employee

### PR DESCRIPTION
**Pull request by daiahnadine**

### CODE REVIEW

File: employee-information-management-system/src/main/java/Main.java [hunk 1]
```diff
@@ -121,5 +124,54 @@ public static void main(String[] args) {
...
+        employee.setTinNo(scanner.nextLine());
+        System.out.print("Pagibig Number: ");
+        employee.setPagibigNo(scanner.nextLine());
+        System.out.print("Employee Phone Number: ");
+        employee.setEmployeeNo(scanner.nextLine());
+
+        employeeFacade.addEmployee(employee);
+
+        scanner.close();
+        Connection con = ConnectionHelper.getConnection();
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/Main.java]: Why do you have ConnectionHelper here?

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java [hunk 2]
```diff
@@ -37,4 +35,33 @@ public boolean updateEmployee(Employee employee) {
         }
         return result;
     }
+
+    // add employee
+    public EmployeeFacadeImpl(EmployeeDao employeeDao) {
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java]: why do constructor dependency injection?

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java [hunk 3]
```diff
@@ -37,4 +35,33 @@ public boolean updateEmployee(Employee employee) {
...
+            if (targetEmployee != null) {
+                throw new RuntimeException("Employee to add already exists.");
+            }
+            result = employeeDao.addEmployee(employee);
+        } catch (Exception e) {
+            throw new RuntimeException(e.getMessage());
+        }
+
+        if (result) {
+            System.out.println("Employee added successfully.");
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java]: move these messages in Facade. leave messages for catching exceptions

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java [hunk 4]
```diff
@@ -37,4 +35,33 @@ public boolean updateEmployee(Employee employee) {
...
+        } else {
+            System.out.println("Failed to add employee.");
+        }
+        return result;
+    }
+    @Override
+    public Employee getEmployeeById(String id) {
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java]: why null again?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java (hunk 3 above)
Comment (daiahnadine) [reply employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java]: Sir di ko po nagets masyado, ano po ibig sabihin nito sir? Thank you po.

**Review by RoCS2024 — APPROVED** [review]

**Review by rosellemarithe — APPROVED** [review]

**Pull request merged**